README.md

```diff
@@ -217,6 +217,14 @@ Client certificates are an alternative to traditional authentication mechanisms.
 
 Newman supports SSL client certificates, via the following CLI options (available with Newman `v3` style `run` only):
 
+#### Use a ssl client certificates configuration file (supports multiple certificates per run)
+
+- --ssl-client-certs<br/>
+The path to the ssl client certs configuration file (json format). See [examples/multiple-ssl-client-certs.json](https://github.com/postmanlabs/newman/blob/develop/examples/multiple-ssl-client-certs.json)
+This option allows multiple ssl client certificiates and different ssl client certificate per url or hostname.
+This option and --ssl-client-cert, --ssl-client-key and --ssl-client-passphrase are exclusives.
```

Comment:
How about merging individual cert config into the cert list?
And the individual certs take priority over the same config present in the list (works as overrides).

Reply:
@codenirvana I had the same idea while implementing and testing this feature.
However the individual certificate would work differently than the list:
the individual cert does not have url prefix parameter, how to determine the priority ?

We can decide to use the individual certificate as:
1. the default certificate, i.e. for any url not matched by the url prefixes defined in the list. 
2. Or to use it as the most priority certificate, however there is no url prefix associated to it, so it would be applied all the time: it's still possible by not passing the certificate list.

I actually tried to implement solution 1. and it's not trivial:
Currently if we set url prefix `*` to the individual cert to make it the default, it actually gets matched first even if there is a more specific url prefix.

So I think if we want to implement 1. we will need to change the priority order of url prefixes matching.
For 2, we can just omit the certificate list.

To make it simpler for now, the 2 options are exclusives: only one can be used